src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");

```

Comment:
The message printed should be more informative and indicate that it's a `Truck` that is working. Consider specifying the type of machine in the message, for example: `Truck [machine name] started its work.`